python/ray/autoscaler/aws/example-multi-node-type.yaml

```
@@ -16,27 +16,26 @@ available_node_types:
     cpu_4_ondemand:
         node_config:
             InstanceType: m4.xlarge
-        resources: {"CPU": 4}
+        # For AWS instances, autoscaler will automatically add the available
+        # CPUS/GPUS/accelerator_type ({"CPU": 4} for m4.xlarge) in "resources".
```

```suggestion
        # CPUs/GPUs/accelerator_type ({"CPU": 4} for m4.xlarge) in "resources".
```

Can you actually make it so this example isn't changed semantically? E.g., keep Custom1 on cpu_16_spot  custom2, etc. below.